MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -232,6 +232,32 @@ The `NumericEntry` control supports customization of the UpDownButton's appearan
 
 ![UpDownButtonTemplate support in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonTemplate.png)
 
+## Up Down Button Alignment
+
+The Up Down Button Alignment feature specifies the alignment of the Up Down button in the NumericEntry controls.  
+•   Left – Both the Spin buttons are positioned at the Left
+•   Right – Both the Spin buttons are positioned at the Right
+•   Both – Spin buttons are positioned at the extreme ends.
+
+N> The default value of this property is `Right`
+{% tabs %}
+{% highlight XAML %}
```

Comment:
Add snippets and output for left and both.

Reply:
Changes are updated